### PR TITLE
Lf main

### PR DESCRIPTION
**Pull request by certik**

### CODE REVIEW

File: tests/well/special.f90 [hunk 1]
```diff
@@ -7,8 +7,6 @@ module special
 use utils, only: stop_error
 use optimize, only: bisect
 implicit none
-private
-public spherical_bessel_jn, spherical_bessel_jn_zeros, SPHY
```
Comment (certik) [inline tests/well/special.f90:11]: https://github.com/lfortran/lfortran/issues/2719

File: tests/well/optimize.f90 [hunk 2]
```diff
@@ -5,8 +5,6 @@ module optimize
 use types, only: dp
 use utils, only: stop_error
 implicit none
-private
-public bisect
```
Comment (certik) [inline tests/well/optimize.f90:9]: https://github.com/lfortran/lfortran/issues/2719

File: tests/pseudopotential/vloc_nonrel_dft.f90 [hunk 3]
```diff
@@ -104,10 +104,10 @@ program vloc_nonrel_dft
...
 
 ! Save the radial grid, density, V_h
-open(newunit(u), file="density.txt", status="replace")
-write(u, "((es23.16, ' ', es23.16, ' ', es23.16))") (R(i), density(i), V_h(i), &
-        i=1, size(R))
-close(u)
+!open(newunit(u), file="density.txt", status="replace")
+!write(u, "((es23.16, ' ', es23.16, ' ', es23.16))") (R(i), density(i), V_h(i), &
+!        i=1, size(R))
+!close(u)
```
Comment (certik) [inline tests/pseudopotential/vloc_nonrel_dft.f90:110]: Fixed in main, but prints to stdout: https://github.com/lfortran/lfortran/issues/2720

File: tests/CMakeLists.txt [hunk 4]
```diff
@@ -1,4 +1,4 @@
-add_subdirectory(mixings)
+#add_subdirectory(mixings)
```
Comment (certik) [inline tests/CMakeLists.txt:1]: Fixed in main.

File: tests/well/optimize.f90 (hunk 2 above)
Comment (Shaikh-Ubaid) [reply tests/well/optimize.f90:9]: https://github.com/lfortran/lfortran/issues/2722, https://github.com/lfortran/lfortran/pull/2723

File: tests/well/special.f90 (hunk 1 above)
Comment (Shaikh-Ubaid) [reply tests/well/special.f90:11]: https://github.com/lfortran/lfortran/issues/2722, https://github.com/lfortran/lfortran/pull/2723

**Comment by certik** [comment]
Closing in favor of https://github.com/certik/dftatom/pull/48.

**Pull request closed without merging**